build/common.sh

```diff
@@ -573,23 +542,47 @@ function kube::build::ensure_data_container() {
     # We want this to run as root to be able to chown, so non-root users can
     # later use the result as a data container.  This run both creates the data
     # container and chowns the GOPATH.
+    #
+    # The data container creates volumes for all of the directories that store
```

Comment:
Do you know how much sympathy I have for "I want to build from a mac" ?  :)

But I am a little confused, I thought the data container only carries /go/src/k8s.io/kubernetes/_output anyway - nothing to do with /usr/local/go (excellent point, though).  Given that you assert that bind mounts are slow, we still need to copy the source code into a container, or the build will be v.slow (only for Mac people, who should be used to that by now anyway).